application/testing/CMakeLists.txt

```diff
@@ -187,7 +192,7 @@ f3d_test(NAME TestVolumeMag DATA vase_4comp.vti ARGS -vb)
 f3d_test(NAME TestVolumeComp DATA vase_4comp.vti ARGS -vb --comp=3)
 f3d_test(NAME TestVolumeDirect DATA vase_4comp.vti ARGS -vb --comp=-2)
 f3d_test(NAME TestVolumeCells DATA waveletArrays.vti ARGS -vb --cells)
-f3d_test(NAME TestVolumeColoringArray DATA waveletArrays.vti ARGS -vb --coloring-array=Result LONG_TIMEOUT)
+f3d_test(NAME TestVolumeColoringArray DATA waveletArrays.vti ARGS -vb --coloring-array=Result LONG_TIMEOUT THRESHOLD 0.07)
```

Comment:
not sure why the UI would impact this tests though

Reply:
I've updated the baseline and it works fine in all configurations now. Not sure why it changes with this PR tho, but I guess that's OpenGL magic